.vsts.release.yml

```diff
@@ -75,10 +75,6 @@ extends:
       - job: Set_variables
         displayName: Set release-specific variables
         steps:
```

Comment:
This is release pipeline, so CI won't help with it.
It way work only if we'll add some e2e pipeline test which will emulate the release, but I'm not sure it's worth it

Reply:
I understand the concern about effort vs payoff.  Could we do a targeted e2e test for issues caused by changing .vsts.release.yml

It will save time 2 ways:
1. It will make it much easier to make changes to .vsts.release.yml
2. It will save the time due dealing with the failures that occurred after merging the break 

Not blocking for this PR

Reply:
Prepared WI: https://dev.azure.com/mseng/AzureDevOps/_workitems/edit/2185444